Resources/Computing/BatchSystems/TimeLeft/SLURMResourceUsage.py

```diff
@@ -53,7 +51,7 @@ def getResourceUsage(self):
       cpu = float(cpu)
 
     consumed = {'CPU': wallClock,
-                'CPULimit': wallClockLimit,
```

Comment:
Oops, I should have added a comment, and I will do it: SLURM time left is based on `WallClock/Limit` and not `CPU/Limit` while DIRAC `TimeLeft` uses `CPU/Limit` values to make the time left computation.
That's why there was `CPULimit: wallClockLimit` (this was a quick solution, but I agree that this is not clear and maybe this should be made differently...)

Reply:
So, this should be changed back? If that's the case, can you do it, while adding a comment?